src/components/sidebar/Stats.jsx

```diff
@@ -64,6 +52,18 @@ class Stats extends Component {
     };
   }
 
+  /**
+   * React component lifecycle method getDerivedStateFromProps
+   * @param {Object} nextProps - received props
+   */
+  static getDerivedStateFromProps(nextProps) {
+    return {
```

Comment:
I am a big fan of destructuring, i.e, `let {title, page, stats} = nextProps` and you can use Object literals to define the key-value pairs with just one liners.